_pages/launching-software/infrastructure.md

```diff
@@ -7,8 +7,11 @@ questions:
 ## Overview
 
 At some point, you're going to want to deploy your system. You have a good idea of the final destination of your system early on
-in the project. **If you're building [a site that has no server-side code](#static-sites), aim to use [Federalist](#federalist). If you're
-going to host server-side code, aim to deploy to [cloud.gov](#cloudgov).** You can also deploy to [TTS-managed infrastructure as a service (IaaS)](#infrastructure-as-a-service-iaas) directly, but your life will be harder. For GSA systems, see [comparison of hosting options](https://docs.google.com/spreadsheets/d/1TTu6R9vKOR5eiyC0tjF5XfaM9ozVp0FMoKDn_ZJOxG8/edit##gid=0).
+in the project.
+
+- **If you're building [a site that has no server-side code](#static-sites), aim to use [Federalist](#federalist).**
+- **If you're going to host server-side code, aim to deploy to [cloud.gov](#cloudgov).** You can also deploy to [TTS-managed infrastructure as a service (IaaS)](#infrastructure-as-a-service-iaas) directly, but your life will be harder. For GSA systems, see [comparison of hosting options](https://docs.google.com/spreadsheets/d/1TTu6R9vKOR5eiyC0tjF5XfaM9ozVp0FMoKDn_ZJOxG8/edit##gid=0).
+- **If [cloud.gov](#cloudgov) doesn't meet your needs, try reaching out to an [advocate](https://docs.google.com/presentation/d/1XxYku-0jrdjYlag_BOvtFGYiwPMAf2iye2H4tWRMaNI/edit) in the Cloud Integration Shared Services (CISS) Division which has established a Cloud Advisory & Enablement framework.** CISS works with FAS Cloud Services closely throughout the Cloud Enablement phases. The journey phases consist of reviewing [requirements](https://docs.google.com/document/d/1xAd8Ot3LymJFZII1b6kG3zgj5ylWDc7Dlo2rgoVFRZg/edit#) and objectives, [analyzing](https://docs.google.com/presentation/d/1_gfszHOCAkWNspyMyvX8ON76hMUNFkZFcxyTxQTBy8Y/edit#slide=id.ge361286607_0_518) and aligning them with [GSA IT vision and shared services](https://sites.google.com/a/gsa.gov/fas_it_playbook/it-playbook), defining cloud transformation strategy for your application and potentially [onboarding](https://forms.gle/L8ePLWtyAqFd42aQ9) your application(s) to FCS Ecosystem.
```

Comment:
@its-a-lisa-at-work can you share this doc with TTS? https://docs.google.com/presentation/d/1XxYku-0jrdjYlag_BOvtFGYiwPMAf2iye2H4tWRMaNI/edit